module2/lessons/setting_up_postico.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: page
+title: Setting Up Postico
+---
+
+After you download and install Postico, when run it for the first time you will see a screen like this:
+
+![Postico Home Screen](./images/postico-1.png)
```

Comment:
Going to bring in this images from the BE repo. We also need to decide if Postico should still be installed during intermission work or if that should be done here. I know you're busy Jamison, but let me know if you have an opinion. If not, Juliette and I will make a decision.